app/domain/logs.rb

```diff
@@ -33,11 +33,6 @@ module Authentication
       code: "CONJ00003D"
     )
 
-    ValidatingAnnotationsWithPrefix = ::Util::TrackableLogMessageClass.new(
```

Comment:
Why you removed this log ? Do we need to update it in some place ?

Reply:
It is not used.
I didn't find any reference to this that needs update.

Reply:
o.k. looks like a lot of those logs not in use, just need to make sure its not written in some document that we need to update as well